ambry-router/src/main/java/com.github.ambry.router/GetBlobInfoOperation.java

```diff
@@ -244,7 +247,17 @@ private void processGetBlobInfoResponse(GetRequestInfo getRequestInfo, GetRespon
       } else {
         getError = getResponse.getPartitionResponseInfoList().get(0).getErrorCode();
         if (getError == ServerErrorCode.No_Error) {
-          handleBody(getResponse.getInputStream());
+          PartitionResponseInfo partitionResponseInfo = getResponse.getPartitionResponseInfoList().get(0);
+          int objectsInPartitionResponse = partitionResponseInfo.getMessageInfoList().size();
```

Comment:
minor. can we name the variable "msgsInPartitionResponse"

Reply:
Done.